packages/render-fidelity-tools/src/artifact-creator.ts

```diff
@@ -301,8 +330,9 @@ export class ArtifactCreator {
       // Ignored...
     }
 
-    const screenshot =
-        await page.screenshot({path: outputPath, omitBackground: true});
+    const screenshot = outputPath ?
+        await page.screenshot({path: outputPath, omitBackground: true}) :
+        await page.screenshot({omitBackground: true});
```

Comment:
Can this be combined using `path: outputPath?` Also, how do you use/override the default value of outputPath? There might be a simpler way to write that function input.